app/assets/javascripts/utils/course_utils.coffee

```diff
@@ -17,8 +20,12 @@ class CourseUtils
 
   # This builds i18n interface strings that vary based on state/props.
   i18n: (message_key, prefix, default_prefix = 'courses') ->
-    key_prefix = prefix
-    key_prefix ||= default_prefix
-    I18n.t("#{key_prefix}.#{message_key}")
+    if prefix
+      message = I18n.t("#{prefix}.#{message_key}")
+
+    if !message || message.startsWith("[missing")
```

Comment:
Aaarg, I take that back! You should use the library's fallback for missing scope, instead:
`I18n.t("some.missing.scope", {defaults: [{scope: "some.existing.scope"}]});`